Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -82,11 +82,11 @@ def get_installation_access_token(installation_id: str, jwt_token: str):
     if response.status_code == 201:
         return response.json()['token']
     elif response.status_code == 403:
-        return_error('403 Forbidden - The credentials are incorrect')
+        return_error('403 Forbidden - The credentials are incorrect')  # noqa: RET503
```

Comment:
why we can't fix the warning instead of ignoring?

Reply:
This ignores the pre-commit [failure](https://github.com/demisto/content/actions/runs/6258965920/job/16994040274)